test/gtest/hvec_set.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
Are you sure you want Barefoot Networks here? 🙂

Reply:
I just copied this from gtest/hvec_map.cpp and tweaked it to use sets instead of maps.  Never clear when/how copyright notices should be updated in that case.